backend/tests/clubs/test_views.py

```diff
@@ -1620,6 +1622,95 @@ def test_club_invite_email_resend(self):
         )
         self.assertIn(resp.status_code, [200, 201], resp.content)
 
+    def test_send_invite_with_acceptance_email(self):
+        self.client.login(username=self.user1.username, password="test")
+
+        # Create a club application
+        now = timezone.now()
+        application = ClubApplication.objects.create(
+            name="Send Acceptance Invivites",
+            club=self.club1,
+            application_start_time=now,
+            application_end_time=now + timezone.timedelta(days=1),
+            result_release_time=now + timezone.timedelta(days=2),
+            is_wharton_council=False,
+        )
+        application.save()
+
+        applicants = [self.user1, self.user2, self.user3, self.user4]
+        # 4 = accepted, 2 = rejected, 1 = pending
```

Comment:
nit: line-specific comments can generally go on the same line as the code they're referring to. I usually hit tab, and then leave the comment there.

Reply:
Following up on this: I'm pretty sure you can use the enums defined in the `ApplicationSubmission` model, instead of their integer values. Something like `ApplicationSubmission.PENDING`, `ApplicationSubmission.REJECTED_AFTER_WRITTEN`, etc.